class02/exercise_5.py

```diff
@@ -49,25 +62,36 @@
 # >>> 150
 
 # code up your solution here
-
-
-
-
+numbers = list(int(num) for num in input("Enter the list numbers separated by ',': ").strip("[").strip("]").strip().split(","))
+for num in numbers:
+    if num <= 150:
+        if num % 5 == 0:
+            print(num)
+        else:
+            continue
+    else:
+        break;
 # 5. Pick one of the questions above and use range() for a different solution
 
 # code up your solution here
 
 
-
-
 # 6. Pick one of the question above and use comprehension for a different solution
 
 # code up your solution here
 
-
+list1 = ["Fizz" if i%3 == 0 and i%5 != 0 else "Buzz" if i%3 != 0 and i%5 == 0 else "FizzBuzz" if i%3 ==0 and i%5 == 0 else i for i in range(51)]
+print(list1)
 
 
 # 7. Pcik one of the questions above and use while loop for a different solution
 
 # code up your solution here
+num = 1500
```

Comment:
current code is an infinite loop, please fix it.

Reply:
num = 1500
while num >= 1500 and num <=2700:
  if num%7 == 0 and num%5 == 0:
    print(num)
    num = num + 1
  else:
    num = num +1
    continue

Reply:
I added "num =num+1" after "else:", problem solved